mtp_api/apps/account/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+from transaction.models import Transaction
+
+
+class FileType(models.Model):
```

Comment:
I think I would prefer to have it defined in a DB table, just as a kind of niceness thing, but I did consider the possibility about accepting the string name in the API and doing a custom lookup based on that. Thoughts?